test/Makefile

```
@@ -0,0 +1,20 @@
+# Compile C files and dump their .data section to binary
+
+CC := mips-linux-gnu-gcc
+OBJCOPY := mips-linux-gnu-objcopy
+NM := mips-linux-gnu-nm
```

```suggestion
CROSS := mips-linux-gnu-

CC := $(CROSS)gcc
OBJCOPY := $(CROSS)objcopy
NM := $(CROSS)nm
```
This suggestion was made by `mips64-ultra-elf-` gang